### PR TITLE
Wizard: LOCALE - Add Languages drop down (HMS-5151)

### PR DESCRIPTION
**Pull request by regexowl**
Rebased on https://github.com/osbuild/image-builder-frontend/pull/2651

This adds a drop down for languages selection. The options are populated with an ouput of `localectl list-locales` and sorted to display results starting with the search term first.

Also added tests.

### CODE REVIEW

File: src/Components/CreateImageWizard/steps/Locale/components/LanguagesDropDown.tsx [hunk 1]
```diff
@@ -1,9 +1,175 @@
...
+      }
+    }
+    setSelectOptions(filteredLanguages.sort((a, b) => sortfn(a, b)));
+
+    // This useEffect hook should run *only* on when the filter value changes.
+    // eslint's exhaustive-deps rule does not support this use.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [filterValue]);
+
+  const sortfn = (a: string, b: string) => {
```
Comment (lucasgarfield) [inline src/Components/CreateImageWizard/steps/Locale/components/LanguagesDropDown.tsx:57]: Need to update this so it uses the reusable utility function. Will take care of it tomorrow.

Comment (regexowl) [reply src/Components/CreateImageWizard/steps/Locale/components/LanguagesDropDown.tsx]: Added a commit for the reusable function, thank you!

**Review by lucasgarfield — APPROVED** [review]
Looks good!

**Pull request merged**